Source/qol/xpbar.cpp

```diff
@@ -34,11 +34,11 @@ void DrawBar(const CelOutputBuffer &out, int x, int y, int width, const ColorGra
 	FastDrawHorizLine(out, x, y + 3, width, gradient[gradient.size() / 2 - 1]);
 }
 
-void DrawEndCap(const CelOutputBuffer &out, int x, int y, int idx, const ColorGradient &gradient)
+void DrawEndCap(const CelOutputBuffer &out, Point point, int idx, const ColorGradient &gradient)

```

Comment:
I would pass `Point` as a const reference, since its size is > than sizeof(int).